<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InGameWidget</class>
 <widget class="QWidget" name="InGameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="frame_game">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QLabel" name="label_compare">
          <property name="text">
           <string>ECompare</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_send">
          <property name="text">
           <string>Guess</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_triesLeft">
          <property name="text">
           <string>Numbre d'essais</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="3">
         <widget class="QLabel" name="label_infosServer">
          <property name="text">
           <string>min, max, nombre d'essais</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>                                                   Number</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_number"/>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_state">
          <property name="text">
           <string>State</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_autoPlay">
       <property name="text">
        <string>AutoPlay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_newGame">
       <property name="text">
        <string>New game</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_finish">
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
